doc/examples/README.rst

```diff
@@ -0,0 +1,138 @@
+and ``benzene/benzene.pdb`` are needed. The path to these files is
+already in the runinput file so leave all files in place.
+
+
+Benzene directory
+-----------------
+
+The ``benzene`` directory contains input files for GROMACS
+
+* ``benzene.pdb``: coordinates in PDB format
+* ``benzene.itp``: topology and parameters
+
+an image of the chemical structure ``benzene.pdb.png``, output from
+plotting dH/dlambda (for the MDPOW TI estimator) ``dVdl.pdf`` for
+benzene in water.
+  
+The  ``README.txt`` file explains how to try out the Python API, with
+the commands recorded in ``session.py``.
+
+
+
+Using the CLI
+=============
+
+Make sure that GROMACS commands can be run (e.g., ``source GMXRC`` or
+``module load gromacs``). All commands are run from the directory that
+contains the run input file ``benzene.yml``.
+
+To calculate the water solvation free energy::
+
+  mdpow-equilibrium --solvent water benzene.yml
+  mdpow-fep --solvent water benzene.yml
+  mdpow-solvationenergy --solvent water benzene
+
+See output in file ``energies.txt`` where all energies are in kJ/mol::
+
+  ITP mol    solvent              dG   err_dG   Coulomb  err_Cou       VDW  err_VDW directory 
+  ---------- ---------------- ------ -------- --------- -------- --------- -------- ---------
+  BNZ        water             -8.32     1.49     +7.53     0.51     +0.78     1.40  benzene
+
+
+To calculate the octanol solvation free energy::
+
+  mdpow-equilibrium --solvent octanol benzene.yml
+  mdpow-fep --solvent octanol benzene.yml
+  mdpow-solvationenergy --solvent octanol benzene
+ 
+The octanol energies are appended to the ``energies.txt`` file::
+
+  BNZ        water             -8.32     1.49     +7.53     0.51     +0.78     1.40  benzene
+  BNZ        octanol          -17.03     1.24     +1.93     0.25    +15.10     1.22  benzene  
+
+Based on these *very* short simulations, the solvation free energy of benzene
+in water of -8.32±1.49 kJ/mol is less favorable than the solvation free energy
+in octanol of –17.03±1.24 kJ/mol.
+
+Using these numbers, one can directly calculate the water-octanol partition
+coefficient as
+
+.. math::
+
+   \log P_{OW} = (\Delta G_W - \Delta G_O)/kT \log e
+
+or use a script ::
+
+  mdpow-pow benzene/
+
+which outputs ::
+
+  mdpow.fep   : INFO     [BNZ] Free energy of transfer water --> octanol: -8.712 (1.941) kJ/mol
+  mdpow.fep   : INFO     [BNZ] log P_ow: 1.517 (0.338)
+
+and appends a line to the file ``pow.txt`` with content ::
+
+
+  ITP mol    transferFE error          logPow err_logPow directory 
+  ---------- --------- ------ ------ -------- ---------- --------- 
+  BNZ         -8.71     1.94  logPow    +1.52     0.34   benzene
+
+(It also writes output to ``energies.txt``. See online docs.)
+
+Instead of running ``mdpow-solvationenergy`` for different solvents, one can
+also just run ``mdpow-pow DIRECTORY`` and it will run the separate free energy
+calculations automatically.
```

Comment:
I would put "automatically, and then compute the octanol-water partition coefficient."

Reply:
added in 0124569845080456f40a02c718dacfe5ebd69c08